### PR TITLE
Bug 1212538 - [Accessibility] Change Add Contact screen reader label …

### PR DESCRIPTION
**Pull request by s-clerc**
…to Add Recipient.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=34957fe50d5e2f77fb2ab951e91e8f55a74ea8d1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8c4b35e2fd3b52ee96aee7f2ef65f3d8bfb1b2b4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4de71d7e209b954988f85b0f505fd5df869808f5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=50fbefab086530df295bb0604c65d4422c29e5c1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0df6b48f299d748694688512fc073c789472c279

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=55218a8c3f7822fb776aeb80d3872f835cc549af

File: apps/email/locales/email.en-US.properties [hunk 1]
```diff
@@ -502,11 +502,11 @@ compose-bcc=bcc
 # screen.
 compose-autocomplete-match.ariaLabel={{name}} {{address}}
 
-# LOCALIZATION NOTE(compose-contact-add): For compose card, a label for an add
+# LOCALIZATION NOTE(compose-contact-add-recipient): For compose card, a label for an add
 # contact buttons rendered as a circled plus sign and used to add to, cc and bcc
 # recipients. The following string is spoken by screen readers and not shown on
-# the screen.
-compose-contact-add.ariaLabel=Add contact
+# the scree
```
Comment (yzen) [inline apps/email/locales/email.en-US.properties]: nit: you lost 'n.'

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ecc55e2ca0ff0060adcde5eae0ec9029e576c867

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ea50e2051a76bc57bfd3f48106ae68fdb6e98f1d

**Pull request merged**